tools/system_libs.py

```diff
@@ -776,6 +776,9 @@ class libc(DebugLibrary, AsanInstrumentedLibrary, MuslInternalLibrary, MTLibrary
              '-Wno-string-plus-int',
              '-Wno-pointer-sign']
 
+  # Include internal proxying header.
+  cflags += [f'-I{utils.path_from_root("system/lib/pthread")}']
```

Comment:
I think you can just do `include = ['system/lib/pthread']` here.

Reply:
Actually `MuslInternalLibrary` already includes this directory in its `includes`, so this should not needed at all.